http-clients/apache-client/src/main/java/software/amazon/awssdk/http/apache/internal/impl/ApacheHttpRequestFactory.java

```diff
@@ -151,14 +150,6 @@ private void addHeadersToRequest(HttpRequestBase httpRequest, SdkHttpRequest req
                 */
                .filter(e -> !IGNORE_HEADERS.contains(e.getKey()))
                .forEach(e -> e.getValue().forEach(h -> httpRequest.addHeader(e.getKey(), h)));
-
-        /* Set content type and encoding */
-        if (httpRequest.getHeaders(HttpHeaders.CONTENT_TYPE) == null ||
```

Comment:
Do we know who added this? Seems weird we included this when it wasn't needed...

Reply:
Nope. It was there in the very first "Developer Preview" commit.